archinstall/lib/user_interaction.py

```diff
@@ -564,24 +654,35 @@ def select_language(options, show_only_country_codes=True):
 		for index, language in enumerate(languages):
 			print(f"{index}: {language}")
 
-		print(" -- You can choose a layout that isn't in this list, but whose name you know --")
-		print(" -- Also, you can enter '?' or 'help' to search for more languages, or skip to use US layout --")
+		print(
+			" -- You can choose a layout that isn't in this list, but whose name you know --"
+		)
+		print(
+			" -- Also, you can enter '?' or 'help' to search for more languages, or skip to use US layout --"
+		)
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability.